src/DisplayPlug.vala

```diff
@@ -46,13 +49,38 @@ public class Display.Plug : Switchboard.Plug {
             displays_view = new DisplaysView ();
             var action_bar = new Gtk.ActionBar ();
             action_bar.get_style_context ().add_class (Gtk.STYLE_CLASS_INLINE_TOOLBAR);
+            
+            detect_touchscreen ();
```

Comment:
I would prefer to have `detect_touchscreen ()` return a boolean value that would be assigned into a temporary variable instead of having an instance member of `Plug` class.